shared/native/notifications.desktop.js

```diff
@@ -1,18 +1,30 @@
 // @flow
 /* global Notification */ // tell lint this exists
-import moment from 'moment'
+import {debounce} from 'lodash'
 
-const rateLimit: {[key: string]: Object} = {}
+const rateLimit: {[key: string]: () => void} = {}
+const rateLimitPayloads: {[key: string]: {title: string, opts: ?Object, onClick: ?() => void}} = {}
```

Comment:
keeping the payloads so we only show the last data on the trailing edge of the debounce